mucommander-translator/src/main/resources/languages.properties

```diff
@@ -30,6 +30,7 @@ sk = Slovenčina
 sl = Slovenščina
 sv = Svenska
 tr = Türkçe
+# btw, Why not UA? UK is United Kingdom, Ukraine is UA! Slava Ukraini!
```

Comment:
:)
I didn't check it on a system with Ukrainian locale, it might be uk_UA but that's how it was contributed..

Reply:
yeah! it is uk_UA to my surprise :) - I will remove this comment in another commit